### ISSUE TITLE
File Not found error - no commands work.

### ISSUE THREAD

**Issue by sweetbbak** [think-biq/piper-whistle#1]
Every command returns file not found error, and no command would populate this file. Is there some step I'm missing here? I'd like to use this tool to search for piper models mainly. Im using Linux.
```
FileNotFoundError: [Errno 2] No such file or directory: '/home/$USER/.config/piper-whistle/index.json'
```
also `pkg_resources==0.0.0` in the requirements.txt doesn't match any pypi packages.

**Comment by BlurryRoots** [comment]
Thanks for reaching out about this. The file not found error appears to have been caused if no database index is present. Was supposed to autodownload, but I messed up the order of checks. Should be fixed with 73313ffbef1562f9b2407cd5b20cfa3fa1f777eb
You may setup the database intially by running ```whistle -vR```

**Comment by sweetbbak** [comment]
I appreciate it!
